### PR TITLE
[SBH-12] Update README.md

### PR DESCRIPTION
**Pull request by PedroSousaAlmeida**
ajdhgwahgdlajhwgdhlgawlgdawgdlawygdlyagwlydgalwlgj

### CODE REVIEW

**Comment by PedroAlmeidaSolutionsbi** [comment]
falta a tarefa do notion

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ dawldajwldh
 d'alwjdalkjdjkadja
 
 dka;whdkjawhdkajhd[](url)
+
+dhawjdhawhdjawh
```
Comment (PedroAlmeidaSolutionsbi) [inline README.md:8]: legal era isso

**Review by PedroAlmeidaSolutionsbi — APPROVED** [review]
perfeito

File: README.md (hunk 1 above)
Comment (PedroSousaAlmeida) [reply README.md:8]: o beleza

**Pull request merged**